nemo/collections/nlp/modules/common/megatron/megatron_perceiver_encoders.py

```diff
@@ -59,9 +59,6 @@ def __init__(
         encoder_attn_mask_type=AttnMaskType.padding,
         hidden_dropout=0.1,
         attention_dropout=0.1,
```

Comment:
What happened to 
```
        position_embedding_type='learned_absolute',
        relative_attention_num_buckets=32,
        relative_attention_max_distance=128,
```
?

Reply:
I've removed RPE support from perceivers for now.